profiles/views_test.py

```diff
@@ -56,8 +56,10 @@
 
 def format_image_expectation(profile):
     """formats a profile image to match what will be in JSON"""
-    profile["image"] = "http://testserver{}".format(profile["image"])
-    profile["image_small"] = "http://testserver{}".format(profile["image_small"])
+    if 'image' in profile:
```

Comment:
Can you change this to a for loop which checks `image` `image_small` and `image_medium`?